### PR TITLE
Update content for reflecting new naming conventions

### PR DESCRIPTION
**Pull request by bjoernricks**
**What**:

We agreed on using the term Greenbone Community Edition for all of our
software.

**Why**:

We have new naming conventions.

**How**:

**Checklist**:

- [ ] Tests
- [ ] CHANGELOG.md entry
- [ ] Documentation

### CODE REVIEW

File: src/architecture.rst [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
 
 .. figure:: images/gvm-architecture.png
-  :alt: GVM Architecture
+  :alt: Greenbone Community Edition Architecture
 
-  Architecture of GVM 21.04
+  Architecture in release 21.04
 
-GVM is grouped into three major parts:
+It is grouped into three major parts:
```
Comment (y0urself) [inline src/architecture.rst:17]:
```
The architecture for the version 21.04 is grouped into three major parts:
```
Maybe like this?

File: src/21.04/index.rst [hunk 2]
```diff
@@ -1,6 +1,6 @@
-==================
-Building GVM 21.04
-==================
+==========================
+Building 21.04 from source
```
Comment (k-schlosser) [inline src/21.04/index.rst:2]:
```suggestion
Building 21.04 from Source
```

Comment (k-schlosser) [reply src/21.04/index.rst:2]: Capizalization in headings

File: src/architecture.rst [hunk 3]
```diff
@@ -1,30 +1,30 @@
...
 * Executable scan application that runs vulnerability tests (VT) against target
   systems
 * Greenbone Vulnerability Manager Daemon (gvmd)
 * Greenbone Security Assistant (GSA) with the Greenbone Security Assistant
   Daemon (gsad)
 
-The GVM framework is released under Open Source licenses as the
-:term:`Greenbone Source Edition (GSE)<GSE>`. By using it, Linux distributions
-can create and provide GVM in the form of installation packages.
+The Greenbone Community Edition is released under Open Source licenses. By using
```
Comment (k-schlosser) [inline src/architecture.rst:25]:
```suggestion
The Greenbone Community Edition is released under open-source licenses. By using
```

File: src/conf.py [hunk 4]
```diff
@@ -17,9 +17,9 @@
 
 # -- Project information -----------------------------------------------------
 
-project = 'Greenbone Documentation'
-copyright = '2021, Greenbone Networks'
-author = 'Greenbone Networks'
+project = "Greenbone Community Documentation"
+copyright = "2021-2022, Greenbone Networks"
```
Comment (k-schlosser) [inline src/conf.py:21]:
```suggestion
copyright = "2021–2022, Greenbone Networks"
```

File: src/faq.md [hunk 5]
```diff
@@ -11,45 +11,52 @@ depth: 2
 
 For a comprehensive background see {doc}`History of OpenVAS <history>`.
 
-## GVM, GSE, GSM TRIAL, GCF, GSF, GSM, GPE,  ... what are these things about?
+## Where can I ask question and get support?
```
Comment (k-schlosser) [inline src/faq.md:14]:
```suggestion
## Where can I ask questions and get support?
```

File: src/faq.md [hunk 6]
```diff
@@ -11,45 +11,52 @@ depth: 2
...
 
-See [Solution Comparison paper](https://www.greenbone.net/wp-content/uploads/Solution_Comparison_EN.pdf)
-and the {doc}`glossary`.
+Support is only provided for the Greenbone Enterprise Appliance products.
```
Comment (k-schlosser) [inline src/faq.md:16]:
```suggestion
Support is only provided for the Greenbone Enterprise product line.
```

File: src/faq.md [hunk 7]
```diff
@@ -11,45 +11,52 @@ depth: 2
...
+Besides that the Greenbone Community can be reached at [https://community.greenbone.net][community forum]
```
Comment (k-schlosser) [inline src/faq.md:17]:
```suggestion
Besides that, the Greenbone Community can be reached at [https://community.greenbone.net][community forum]
```

File: src/faq.md [hunk 8]
```diff
@@ -11,45 +11,52 @@ depth: 2
...
+to ask question. In this forum several Greenbone developers trying to help on a
```
Comment (k-schlosser) [inline src/faq.md:18]:
```suggestion
to ask question. In this forum, several Greenbone developers trying to help on a
```

File: src/faq.md [hunk 9]
```diff
@@ -11,45 +11,52 @@ depth: 2
...
-commercial Greenbone OS. If you have installed GVM software from your
-distribution, an external package repository or even a docker image Greenbone
-wasn't involved in providing this installation method to you. The software from
-these sources may be heavily adjusted, outdated or even completely broken.
-Therefore if you have issues with the software please contact the provider of
-the packages first and create a ticket at their issue tracker. How to contact
-the provider really depends and varies.
+commercial Greenbone OS. If you have installed the Greenbone Community Edition
+software from your distribution like Kali Linux, an external package repository
+or even some random docker image Greenbone wasn't involved in providing this
```
Comment (k-schlosser) [inline src/faq.md:28]:
```suggestion
or even some random docker image, Greenbone was not involved in providing this
```

File: src/faq.md [hunk 10]
```diff
@@ -11,45 +11,52 @@ depth: 2
...
+installation method to you.
+
+The development of the Greenbone Community Edition and packaging for a
+distribution or container image is completely independent. Therefore our
+software provided from these sources may be heavily adjusted, outdated or even
+completely broken. If you have issues with the software please contact the
```
Comment (k-schlosser) [inline src/faq.md:34]:
```suggestion
completely broken. If you have issues with the software, please contact the
```

File: src/faq.md [hunk 11]
```diff
@@ -11,45 +11,52 @@ depth: 2
...
+provider of the packages first and create a ticket at their issue tracker. How
+to contact the provider really depends and varies.
```
Comment (k-schlosser) [inline src/faq.md:36]:
```suggestion
to contact the provider depends and varies.
```

File: src/faq.md [hunk 12]
```diff
@@ -11,45 +11,52 @@ depth: 2
...
 Greenbone can't provide installation docs for the many available Linux
 distributions available. Setups, configurations, build systems, packaging tools,
 available software, best practices, ... diverge to much to offer *official*
-packages, scripts or anything else from Greenbone. We offer this installation
-documentation and the [community forum](https://community.greenbone.net)
-for discussing setup problems. Therefore Greenbone also doesn't recommend any
-distribution nor do we prefer one distribution over the other. Most of the time
-we are even not aware which distribution ships which version of our software.
+packages, scripts or anything else from Greenbone. We offer this Greenbone Community
+documentation and the [community forum] for discussing setup problems. Therefore
```
Comment (k-schlosser) [inline src/faq.md:49]:
```suggestion
documentation and the [community forum] for discussing setup problems. Therefore,
```

File: src/faq.md [hunk 13]
```diff
@@ -11,45 +11,52 @@ depth: 2
...
+Greenbone also doesn't recommend any distribution nor do we prefer one
+distribution over the other. Most of the time we are even not aware which
```
Comment (k-schlosser) [inline src/faq.md:51]:
```suggestion
distribution over the other. Most of the time, we are even not aware which
```

File: src/faq.md [hunk 14]
```diff
@@ -11,45 +11,52 @@ depth: 2
...
 
-## Which release contains which component? GOS version vs. GSE version?
+## Which release contains which component? GOS version vs. GVM version?
 
 It was often confusing to find out which software component of the
-{term}`Greenbone Source Edition <GSE>` belongs to which GVM release.
-Additionally the [Greenbone OS](https://community.greenbone.net/t/about-the-greenbone-os-gos-category/214)
-used in the [GSM Trial Virtual Machine](https://community.greenbone.net/t/setting-up-the-greenbone-security-manager-trial-gsm-trial-virtual-machine/6939)
+{term}`Greenbone Community Edition` belongs to which GVM release.
+Additionally the {term}`Greenbone OS <GOS>`
```
Comment (k-schlosser) [inline src/faq.md:58]:
```suggestion
Additionally, the {term}`Greenbone OS <GOS>`
```

File: src/faq.md [hunk 15]
```diff
@@ -11,45 +11,52 @@ depth: 2
...
+used in the [Greenbone Enterprise TRIAL Virtual Machine](https://www.greenbone.net/en/testnow/)
 had a different versioning scheme then GVM. We are aware of this and therefore
```
Comment (k-schlosser) [inline src/faq.md:60]:
```suggestion
had a different versioning scheme than GVM. We are aware of this and therefore
```

File: src/faq.md [hunk 16]
```diff
@@ -11,45 +11,52 @@ depth: 2
...
 with the 20.08 release we changed our versioning scheme to
```
Comment (k-schlosser) [inline src/faq.md:61]:
```suggestion
with the 20.08 release, we changed our versioning scheme to
```

File: src/glossary.rst [hunk 17]
```diff
@@ -29,63 +29,67 @@ Glossary
...
-    The source edition is adopted by external third parties, e.g., if the
+    The :term:`Greenbone Community Edition` consists of several services. This
+    software framework has been released as Greenbone Vulnerability Management (GVM).
+    It is developed as part of the commercial :term:`Greenbone Enterprise`
+    product line. Sometimes referred as the OpenVAS framework.
+
+  Greenbone Community Edition
+    The Greenbone Community Edition covers the actual releases of the Greenbone
+    application stack for vulnerability scanning and vulnerability management
+    :term:`GVM` provided as Open Source software to the community.
```
Comment (k-schlosser) [inline src/glossary.rst:68]:
```suggestion
    :term:`GVM` provided as open-source software to the community.
```

File: src/glossary.rst [hunk 18]
```diff
@@ -29,63 +29,67 @@ Glossary
...
-    <https://community.greenbone.net/t/about-the-greenbone-professional-edition-category/146>`_
-    is the Greenbone product line for on-premises solutions.
-    Included are virtual or hardware :term:`Greenbone Security Manager (GSM) <GSM>`
-    appliances with the :term:`Greenbone Operating System (GOS)<GOS>`,
+    the Greenbone Community Edition.
+
+  Greenbone Enterprise
+    Greenbone Enterprise is the `Greenbone product line for on-premises solutions
+    <https://www.greenbone.net/en/product-comparison/>`_.
+    Included are virtual or hardware Greenbone Enterprise appliances with the
```
Comment (k-schlosser) [inline src/glossary.rst:76]:
```suggestion
    Included are virtual or hardware Greenbone Enterprise Appliances with the
```

File: src/glossary.rst [hunk 19]
```diff
@@ -29,63 +29,67 @@ Glossary
...
+    :term:`Greenbone Operating System (GOS)<GOS>`,
     the :term:`Greenbone Vulnerability Management (GVM) framework <GVM>`, and
-    the :term:`Greenbone Security Feed (GSF) <GSF>`.
+    the :term:`Greenbone Enterprise Feed`.
 
-  GCF
-    The Greenbone Community Feed (GCF) is the freely available feed for vulnerability
+  Greenbone Community Feed
+    The Greenbone Community Feed is the freely available feed for vulnerability
     information licensed as Open Source. It contains basic scan configurations,
```
Comment (k-schlosser) [inline src/glossary.rst:83]:
```suggestion
    information licensed as open source. It contains basic scan configurations,
```

File: src/history.rst [hunk 20]
```diff
@@ -73,41 +79,43 @@ of thousands of lines of code were produced and there was almost no day without
...
-former service :term:`openvassd` was turned into a command-line tool
-:term:`openvas`. It is controlled by the service layer :term:`ospd-openvas`.
+GVM-11 introduced substantial architectural changes: the former service
+:term:`openvassd` was turned into a command-line tool :term:`openvas`. It is
+controlled by the service layer :term:`ospd-openvas`.
 This concept essentially replaces the old stateful, permanent and proprietary
 OTP (OpenVAS Transfer Protocol) by the new stateless, request-response
 XML-based and generic :term:`OSP (Open Scanner Protocol)<OSP>`.
+
+With GVM 20.08 the release versioning changed to `Calendar Versioning <https://calver.org/>`_.
```
Comment (k-schlosser) [inline src/history.rst:115]:
```suggestion
With GVM 20.08, the release versioning changed to `Calendar Versioning <https://calver.org/>`_.
```

File: src/glossary.rst (hunk 19 above)
Comment (bjoernricks) [inline src/glossary.rst:83]:
```suggestion
    information licensed as open-source. It contains basic scan configurations,
```

File: src/index.rst [hunk 21]
```diff
@@ -1,30 +1,29 @@
-Greenbone Source Edition – Documentation
-========================================
+Greenbone Community – Documentation
+===================================
```
Comment (bjoernricks) [inline src/index.rst:2]:
```suggestion
Greenbone Community Edition – Documentation
===========================================
```

File: src/faq.md [hunk 22]
```diff
@@ -73,10 +80,9 @@ to exchange experiences and problems about a [Free Software](https://en.wikipedi
 project and not to get instant advises from the developers or professional
 support to fix your current issue.
 
-If you are using a self-compiled version of our GVM stack not build from this
+If you are using a self-compiled version of our software stack not build from this
```
Comment (y0urself) [inline src/faq.md:83]:
```suggestion
If you are using a self-compiled version of the Greenbone software stack not build from this
```

File: src/faq.md [hunk 23]
```diff
@@ -11,47 +11,54 @@ depth: 2
 
 For a comprehensive background see {doc}`History of OpenVAS <history>`.
 
-## GVM, GSE, GSM TRIAL, GCF, GSF, GSM, GPE,  ... what are these things about?
+## Where can I ask questions and get support?
 
-See [Solution Comparison paper](https://www.greenbone.net/wp-content/uploads/Solution_Comparison_EN.pdf)
-and the {doc}`glossary`.
+Support is only provided for the Greenbone Enterprise product line.
+Besides that, the Greenbone Community can be reached at [https://community.greenbone.net][community forum]
```
Comment (y0urself) [inline src/faq.md:17]:
```suggestion
Besides that, the Greenbone Community can be reached at the [https://community.greenbone.net][community forum]
```

**Review by y0urself — CHANGES_REQUESTED** [review]

**Review by y0urself — APPROVED** [review]

**Pull request merged**